Makefile

```diff
@@ -30,13 +30,13 @@ $(os):
 	mkdir -p -m 0750 dist/${@}-x86_64/var/cache/zps
 	mkdir -p -m 0750 dist/${@}-x86_64/var/tmp/zps
 	mkdir -p -m 0755 dist/${@}-x86_64/usr/share/zps/certs/zps.io
-	cp ../zps.io/ca.pem ../zps.io/intermediate.pem ../zps.io/zps.pem dist/${@}-x86_64/usr/share/zps/certs/zps.io
+	#cp ../zps.io/ca.pem ../zps.io/intermediate.pem ../zps.io/zps.pem dist/${@}-x86_64/usr/share/zps/certs/zps.io
```

Comment:
I'm curious - why was this changed? It seems unrelated.

Reply:
changed it for local development, reverted! thanks